backend/src/test/java/gov/cdc/usds/simplereport/api/queue/QueueMutationResolverTest.java

```diff
@@ -0,0 +1,95 @@
+package gov.cdc.usds.simplereport.api.queue;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import gov.cdc.usds.simplereport.db.model.DeviceSpecimenType;
+import gov.cdc.usds.simplereport.db.model.DeviceType;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.db.model.auxiliary.DiseaseResult;
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+import gov.cdc.usds.simplereport.service.BaseServiceTest;
+import gov.cdc.usds.simplereport.service.DeviceTypeService;
+import gov.cdc.usds.simplereport.service.PersonService;
+import gov.cdc.usds.simplereport.service.TestOrderService;
+import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration;
+import gov.cdc.usds.simplereport.test_util.TestDataFactory;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@SliceTestConfiguration.WithSimpleReportStandardUser
+class QueueMutationResolverTest extends BaseServiceTest<PersonService> {
+  @Autowired private TestDataFactory _dataFactory;
+
+  private UUID _patientId;
+  private DeviceType _deviceType;
+  private DeviceSpecimenType _deviceSpecimenType;
+  private final Date _dateTested = new Date();
+
+  @BeforeEach
+  void setup() {
+    Organization _org = _dataFactory.createValidOrg();
+    _patientId = _dataFactory.createMinimalPerson(_org).getInternalId();
+    _deviceType = _dataFactory.getGenericDevice();
+    _deviceSpecimenType = _dataFactory.getGenericDeviceSpecimen();
+  }
+
+  @SneakyThrows
```

Comment:
If I remove `@SneakyThrows`, I get an error:
`unreported exception NumberParseException; must be caught or declared to be thrown` on `line 61`

Reply:
Hmm, what happens if you add `throws NumberParseException` to the method signature? Like this:

`void getDeviceSpecimenTypeId_callsGetDeviceSpecimenTypeId_whenNoDeviceSpecimenType() throws NumberParseException { ... }`

Reply:
Oh!! that works! Thank you!